tools/release/backport_pr.sh

```diff
@@ -24,59 +24,100 @@ ensure_command () {
   fi
 }
 
+display_usage () {
+  echo "USAGE: $0 PR_NUMBER GITHUB_USER BACKPORT_BRANCHES REVIEWERS [-c PER_BACKPORT_COMMAND]" >/dev/stderr
+  echo "   PR_NUMBER: The number for the PR to be backported." >/dev/stderr
+  echo "   GITHUB_USER: Your GitHub username." >/dev/stderr
+  echo "   BACKPORT_BRANCHES: A space-separated list of branches to which to backport." >/dev/stderr
```

Comment:
I'm not a language lawyer, so while I _think_ this is grammatically correct, it's a bit hard to parse. This is the list of branches that will receive the backported PR, yes?

Reply:
Fair enough. I was a bit skeptical of it when I gave it a second look. I've gone back through the help text with an eye toward wordsmithing. Hopefully it's a bit more understandable now.